src/test/java/org/gridsuite/modification/server/modifications/LoadScalingModificationTest.java

```diff
@@ -0,0 +1,369 @@
+                .andExpect(status().is5xxServerError())
+                .andReturn();
+
+        assertEquals("LOAD_SCALING_ERROR : All filters contains equipments with wrong ids", response.getResponse().getContentAsString());
+    }
+
+    @Test
+    public void testWithWrongFilter() throws Exception {
+        var filter = FilterInfos.builder()
+                .id(FILTER_NOT_FOUND_ID)
+                .name("filter 1")
+                .build();
+
+        var variation = ScalingVariationInfos.builder()
+                .variationMode(VariationMode.PROPORTIONAL)
+                .variationValue(100D)
+                .filters(List.of(filter))
+                .build();
+
+        ModificationInfos modificationToCreate = LoadScalingInfos.builder()
+                .uuid(LOAD_SCALING_ID)
+                .date(ZonedDateTime.now())
+                .type(ModificationType.LOAD_SCALING)
+                .variationType(VariationType.DELTA_P)
+                .variations(List.of(variation))
+                .build();
+
+        String modificationToCreateJson = mapper.writeValueAsString(modificationToCreate);
+
+        var result = mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is5xxServerError()).andReturn().getResponse().getContentAsString();
+        assertEquals(new NetworkModificationException(NetworkModificationException.Type.LOAD_SCALING_ERROR, "404 Not Found: [no body]").getMessage(), result);
+    }
+
+    @Test
+    public void testVentilationModeWithoutDistributionKey() throws Exception {
+        var filter = FilterInfos.builder()
+                .id(FILTER_NO_DK)
+                .name("filter")
+                .build();
+
+        var variation1 = ScalingVariationInfos.builder()
+                .variationValue(100D)
+                .variationMode(VariationMode.VENTILATION)
+                .filters(List.of(filter))
+                .build();
+
+        ModificationInfos modificationToCreate = LoadScalingInfos.builder()
+                .uuid(LOAD_SCALING_ID)
+                .date(ZonedDateTime.now())
+                .type(ModificationType.LOAD_SCALING)
+                .variationType(VariationType.DELTA_P)
+                .variations(List.of(variation1))
+                .build();
+
+        String modificationToCreateJson = mapper.writeValueAsString(modificationToCreate);
+
+        var result = mockMvc.perform(post(getNetworkModificationUri()).content(modificationToCreateJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is5xxServerError()).andReturn().getResponse().getContentAsString();
+        assertEquals(new NetworkModificationException(NetworkModificationException.Type.LOAD_SCALING_ERROR, "This mode is available only for equipment with distribution key").getMessage(), result);
+    }
+
+    @Test
+    public void testScalingCreationWithWarning() throws Exception {
```

Comment:
?

Reply:
we can create scaling succesfully but we have some warning to indicate that the scaling is applied on 2 filters of 3 given filters for example (one of the filters is wrong)